javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Currency calculator</h1>
+    <div class="formOne">
+      <select name="exchange_rate" class="currency"></select>
+      <input type="number" class="fromInput" />
```

Comment:
You should put `min="0"` onn this input. Now it allows to put in negative numbers :)

Reply:
min = 0 helps only if we selecting numbers with arrows in input. Probably have to create if value < 0 in js